spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-repository-sample/src/main/java/com/example/SpannerRepositoryExampleDriver.java

```diff
@@ -26,48 +26,32 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Profile;
 
 /**
  * Application to execute the sample app for Spanner.
  *
  * @author Daniel Zou
  */
 @SpringBootApplication
-public class SpannerExampleDriver {
+public class SpannerRepositoryExampleDriver {
 
```

Comment:
Just a thought, what about simplifying this and combine `SpannerRepositoryExampleDriver.java` and  `SpannerRepositoryExample.java`?
 Like in datastore samples: [DatastoreRepositoryExample.java ](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/ad8cac040f8d8853996215c5a7a7ed022d5ced82/spring-cloud-gcp-samples/spring-cloud-gcp-data-datastore-sample/src/main/java/com/example/DatastoreRepositoryExample.java#L53-L52)